src/chrome/content/rules/ExtremeTech.com.xml

```diff
@@ -0,0 +1,32 @@
+<!--
+	Invalid certificate:
+		- discuss.extremetech.com
+		- metrics.extremetech.com, equivalent to extremetech-com.122.2o7.net
+		- shop.extremetech.com
+
+	Timed out:
+		- extremetech.com, equivalent to www.extremetech.com
+		- mailing.enews.extremetech.com
+-->
+
+<ruleset name="ExtremeTech.com">
+	<target host="extremetech.com" />
+	<target host="www.extremetech.com" />
+	<target host="bbstatic.extremetech.com" />
+	<target host="metrics.extremetech.com" />
+	<target host="product.extremetech.com" />
+	<target host="product2.extremetech.com" />
+	<target host="product3.extremetech.com" />
+	<target host="product4.extremetech.com" />
+	<target host="static.extremetech.com" />
+
+	<securecookie host="^www\.extremetech\.com$" name=".+" />
+
+	<rule from="^http://extremetech\.com/"
+		  to="https://www.extremetech.com/" />
+
+	<rule from="^http://metrics\.extremetech\.com/"
+		  to="https://extremetech-com.122.2o7.net/" />
```

Comment:
Please add a test url for it.

Reply:
I can't find any requests for `metrics.extremetech.com`, so I can't add tests. Removing.

Reply:
See e.g. `http://metrics.extremetech.com/1x1.gif`

Reply:
Adding that, thanks.

Reply:
That actually 404s on `extremetech-com.122.2o7.net`, so removing the target entirely.